platform-configs/protectli-vp4650.robot

```diff
@@ -346,6 +353,18 @@ Flash Protectli VP4650 External
     END
     Should Contain    ${flash_result}    VERIFIED
 
+Flash Protectli VP4650 Internal
```

Comment:
Is it really specific to VP4650? Is not the internal flash procedure the same across devices? We could reuse some generic keyword here, maybe placed in the flash.robot?

Reply:
Platform-specific keyword [removed](https://github.com/Dasharo/open-source-firmware-validation/pull/139/commits/a5fd21c6336e648f60d1f289465308df29cf8601#diff-b796784c908d965f4825f2852e977faa304581bce28ead97ad9c657ff9ac1652L355) and a generic keyword [added](https://github.com/Dasharo/open-source-firmware-validation/pull/139/commits/a5fd21c6336e648f60d1f289465308df29cf8601#diff-80c8f3021210429381ecd1783ac7615d73eafe8cccfd238ff06c3764f5c6e685R175)